internal/data/common.go

```diff
@@ -56,6 +56,10 @@ type AttributeValue struct {
 	orig *otlpcommon.AttributeKeyValue
 }
 
+func NewAttributeValue() AttributeValue {
```

Comment:
Should this be named `NilAttributeValue()`? Maybe a constant since we don't allow set/get on this?

Reply:
Sounds good.